### ISSUE TITLE
maskrcnn_benchmark.trainer INFO print twice when training on my own dataset

### ISSUE THREAD

**Issue by wjp0408** [facebookresearch/maskrcnn-benchmark#134]
## ❓ Questions and Help
**When i training my own dataset,** **the trainer  printed information twice...**
![default](https://user-images.githubusercontent.com/34024933/48250576-c4c03600-e439-11e8-921c-c23bf167ba5f.png)
 **But when i turn back to COCO dataset, it printed properly...**
![default](https://user-images.githubusercontent.com/34024933/48250700-25e80980-e43a-11e8-882d-31ea9db61d46.png)
**I set DATALOADER.NUM_WORKERS= 0 in both config files**,and the only difference between two files is train and test dataset name...
coco:
    TRAIN: ("coco_2014_train", "coco_2014_valminusminival",)
    TEST: ("coco_2014_minival",)
my own dataset(composed of three sub datasets):
    TRAIN: ("a_train", b_train","c_train")
    TEST: ("a_test","b_test","c_test")

and sub-dataset A done as follows:
  class ADataset(torch.utils.data.Dataset):
    def __init__(self, ......): # some dirs

    def __getitem__(self, idx):
        ......
       boxlist = BoxList(boxes, image.size, mode="xyxy")
       boxlist.add_field("labels", labels)
       boxlist.add_field("masks", masks)
       if self.transforms:
            image, boxlist = self.transforms(image, boxlist)
       return image, boxlist, idx

    def get_img_info(self, idx):
       ......
       return {"height": img_height, "width": img_width}

I don't know why this happened... **And when i use only one sub-dataset, it still happened...**
Do i need to add something like "img_ids" to Dataset class?...

**What should I do to solve this problem? Thanks for your help!**

**Comment by fmassa** [comment]
Hi,

This is weird. How many GPUs are you using for training?
Also, is the only change that you did in the code the dataset one? Or are there other changes?

**Comment by wangjp0408** [comment]
**Thanks for your reply !**  
And i think i've got the reason ...
When i created my sub dataset files, i  **import logging , and use "logging.info" to print some informations**. And when i change "logging.info" to "print" in dataset file,  Everything is back to normal...
![image](https://user-images.githubusercontent.com/39023793/48260816-e03a3980-e457-11e8-9b13-9f4a32b2b5b8.png)
Sorry to interrupt.  ：）

**Comment by fmassa** [comment]
Great!
I'm closing the issue, but let us know if you have further questions.
